index.html

```diff
@@ -0,0 +1,235 @@
+                </div>
+            </div>
+
+            <div class="card-works">
+                <div class="image-cover">
+                    <img src="./assets/images/1.png" alt="">
+                </div>
+                <div class="project-info">
+                    <h1>Tonic</h1>
+                    <ul class="label">
+                        <li>CANOPY</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="tags">
+                        <li><a href="">html</a></li>
+                        <li><a href="">css</a></li>
+                        <li><a href="">ruby</a></li>
+                    </ul>
+                    <p><button>See Projects</button></p>
+                </div>
+            </div>
+
+            <div class="card-works">
+                <div class="image-cover">
+                    <img src="./assets/images/2.png" alt="">
+                </div>
+                <div class="project-info">
+                    <h1>Tonic</h1>
+                    <ul class="label">
+                        <li>CANOPY</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>I'am a Software Developer
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti et id alias cumque ducimus
+                        quasi</p>
+                    <ul class="tags">
+                        <li><a href="">html</a></li>
+                        <li><a href="">css</a></li>
+                        <li><a href="">ruby</a></li>
+                    </ul>
+                    <p><button>See Projects</button></p>
+                </div>
+            </div>
+
+            <div class="card-works">
+                <div class="image-cover">
+                    <img src="./assets/images/3.png" alt="">
+                </div>
+                <div class="project-info">
+                    <h1>Tonic</h1>
+                    <ul class="label">
+                        <li>CANOPY</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>I'am a Software Developer
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti et id alias cumque ducimus
+                        quasi</p>
+                    <ul class="tags">
+                        <li><a href="">html</a></li>
+                        <li><a href="">css</a></li>
+                        <li><a href="">ruby</a></li>
+                    </ul>
+                    <p><button>See Projects</button></p>
+                </div>
+            </div>
+        </section>
```

Comment:
- [ ] on this section kindly consider sticking to the design provided
-your design 👇 
![uu](https://user-images.githubusercontent.com/60146030/219660567-0369a96b-48cd-4d31-b90b-4a7a48732955.PNG)
- provided design 👇 
![nk](https://user-images.githubusercontent.com/60146030/219660872-88089c99-ee8a-47f8-840f-66466b2511e1.PNG)
kindly consider displaying one card along